controllers/chat/ChatController.js

```diff
@@ -12,6 +12,7 @@ const UsersFunction = require('../../databases/functions/users');
 const BetterSocialConstantListUtils = require('../../services/bettersocial/constantList/utils');
 const {CHANNEL_TYPE} = require('../../helpers/constants');
 const ChatAnonUserInfoFunction = require('../../databases/functions/chatAnonUserInfo');
+const BetterSocialCore = require('../../services/bettersocial');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `BetterSocialCore` module seems to be a good change, as it is used later in the code for updating channel members. This change improves modularity by separating the logic for updating channel members into its own module.
```diff
+ const BetterSocialCore = require('../../services/bettersocial');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->